hasktorch/src/Torch/Serialize.hs

```diff
@@ -3,9 +3,22 @@ module Torch.Serialize where
 import qualified Torch.Internal.Managed.Serialize as S
 import Torch.Tensor
 import Torch.Internal.Cast
+import Torch.NN
+import Torch.Autograd
 
 save :: [Tensor] -> FilePath -> IO ()
 save inputs file = cast2 S.save inputs file
 
 load :: FilePath -> IO [Tensor]
 load file = cast1 S.load file
+
+saveWts :: Parameterized f => f -> FilePath -> IO ()
+saveWts model filePath = do 
+    let wts =  map (toDependent) (flattenParameters model)
+    save wts filePath
+
+loadSavedWts :: Parameterized b => b -> FilePath -> IO b
```

Comment:
_Weights_ is a very specific term used for only some parameters. Can we please call those functions `saveWeights` and `loadWeights`?

Reply:
Thank you for pointing out, I am at fault, they should be better-called parameters I will fix that.

Reply:
@apaszke I believe you meant to say something along the line of `saveParams` and `loadParams` right?
Also, can you shed some light on the earlier thread?

> @austinvhuang I took a look at torchscript example earlier and from what I understood we can load pre-trained models and run inference in hasktorch but fine-tuning and transfer learning doesn't seem feasible via torchscript and I think that will greatly reduce the models' usability.

Reply:
Oh yes, good point! I meant `saveParameters` and `loadParameters`. Sorry for the mixup